### PR TITLE
Fake html body

### PR DESCRIPTION
**Pull request by rudkjobing**

### CODE REVIEW

File: src/Faker/Provider/HtmlLorem.php [hunk 1]
```diff
@@ -66,6 +66,36 @@ public function randomHtml($maxDepth = 4, $maxWidth = 4)
...
+        return $document->saveHTML();
+    }
+
+    /**
+     * @param int $maxDepth
+     * @param int $maxWidth
+     *
+     * @return string
+     */
+    public function randomBodyFragments($maxDepth = 4, $maxWidth = 4)
```
Comment (WillGibson) [inline src/Faker/Provider/HtmlLorem.php]: Could this be `randomHTMLFragments`?

File: test/Faker/Provider/HtmlLoremTest.php [hunk 2]
```diff
@@ -20,10 +20,27 @@ public function testProvider()
     public function testRandomHtmlReturnsValidHTMLString(){
         $faker = new Generator();
         $faker->addProvider(new HtmlLorem($faker));
-        $node = $faker->randomHtml(6, 10);
+        $node = $faker->randomHtml(5, 5);
         $dom = new \DOMDocument();
         $error = $dom->loadHTML($node);
```
Comment (WillGibson) [inline test/Faker/Provider/HtmlLoremTest.php]: I think `$error` should be `$isValidHtml` like in the test below.

**Comment by WillGibson** [comment]
@rudkjobing Very odd, all the other PRs are doing Travis builds, why not this one? I don't think it's anything to do with this PR or the code in it.

File: src/Faker/Provider/HtmlLorem.php (hunk 1 above)
Comment (WillGibson) [inline src/Faker/Provider/HtmlLorem.php]: I have done this in my fork

File: src/Faker/Provider/HtmlLorem.php [hunk 3]
```diff
@@ -129,7 +159,8 @@ private function addRandomAttribute(\DOMElement $node)
                 $node->setAttribute("class", $this->generator->word);
                 break;
             case 2:
-                $node->setAttribute("id", (string)$this->idGenerator->randomNumber(5));
+            	$randomNumber = $this->generator->randomNumber(5);
```
Comment (WillGibson) [inline src/Faker/Provider/HtmlLorem.php]: It was tabs not spaces. Done in my fork.

File: test/Faker/Provider/HtmlLoremTest.php (hunk 2 above)
Comment (WillGibson) [inline test/Faker/Provider/HtmlLoremTest.php]: Done in my fork.

**Comment by WillGibson** [comment]
@rudkjobing Latest update is, that I think I'm on top of it, but the builds are only working PHP 5.6+. I've just installed 5.3.29 with PHPBrew to see what that's about.

**Comment by WillGibson** [comment]
Hi @rudkjobing I have a PR with passing builds here https://github.com/fzaninotto/Faker/pull/1159.

If you're happy to roll with that, please could you kill this PR to avoid confusion.

Thanks,

Will

**Comment by WillGibson** [comment]
Thanks @rudkjobing I just corrected the link above to my PR.

**Comment by rudkjobing** [comment]
@WillGibson you are very welcome. Thanks for picking up my slack :-)

**Pull request closed without merging**